src/main/java/uk/gov/hmcts/reform/civil/model/docmosis/sealedclaim/ResponseRepaymentDetailsForm.java

```diff
@@ -145,11 +148,26 @@ private static void partAdmissionData(CaseData caseData, ResponseRepaymentDetail
         if (caseData.getSpecDefenceAdmittedRequired() == YesOrNo.YES) {
             alreadyPaid(caseData, builder);
         } else {
-            addRepaymentMethod(
-                caseData,
-                builder,
-                MonetaryConversions.penniesToPounds(caseData.getRespondToAdmittedClaimOwingAmount())
-            );
+            if (useRespondent2(caseData)) {
```

Comment:
maybe extract this to a method for better readability?

Reply:
agreed. The code is the same, it only changes what field is used for the amount

Reply:
Is this what we have in mind?


```java
    private static void partAdmissionData(CaseData caseData, ResponseRepaymentDetailsFormBuilder builder) {
        addDetailsOnWhyClaimIsRejected(caseData, builder);
        if (caseData.getSpecDefenceAdmittedRequired() == YesOrNo.YES) {
            alreadyPaid(caseData, builder);
        } else {
             BigDecimal amountInPennies =
                useRespondent2(caseData) ? caseData.getRespondToAdmittedClaimOwingAmount2() :
                    caseData.getRespondToAdmittedClaimOwingAmount();

            addRepaymentMethod(
                caseData,
                builder,
                MonetaryConversions.penniesToPounds(amountInPennies)
            );
        }
    }
```